src/MiniProfiler.Shared/Timing.cs

```diff
@@ -77,28 +78,76 @@ public Timing(MiniProfiler profiler, Timing parent, string name, decimal? minSav
         /// Gets or sets All sub-steps that occur within this Timing step. Add new children through <see cref="AddChild"/>
         /// </summary>
         [DataMember(Order = 5)]
-        public List<Timing> Children { get; set; }
+        public List<Timing> Children
```

Comment:
I doubt the overhead of the duplication would be that high. The current code is using List<Timing> to keep children and while children are added it is doing similar duplication internally potentially multiple time. This would be just one more duplication. And it is extremely fast (less than a nanosecond for small lists (less than 20 elements) in my tests)